src/ui/units/ql/containers/QL/QL.tsx

```diff
@@ -39,20 +45,52 @@ interface QLProps {
 
 type QLInnerProps = QLProps &
     RouteComponentProps<{qlEntryId?: string; workbookId?: string}> &
-    DispatchProps;
+    DispatchProps & {
+        hotkeysContext?: HotkeysContextType;
+    };
 
 class QL extends React.PureComponent<QLInnerProps> {
     entryDialoguesRef: React.RefObject<EntryDialogues>;
+    private hotkeysAreaRef: React.RefObject<HTMLDivElement>;
 
     constructor(props: QLInnerProps) {
         super(props);
 
         this.entryDialoguesRef = React.createRef();
+        this.hotkeysAreaRef = React.createRef();
+
+        this.props.initEditHistoryUnit({
+            unitId: QL_EDIT_HISTORY_UNIT_ID,
+            setState: ({state}) => {
+                const {ql, wizard} = state as unknown as {ql: QLState; wizard: WizardGlobalState};
+
+                this.props.setQLStore({
+                    store: ql,
+                });
+
```

Comment:
QL uses Wizard's store part (visualization), so we need to set both stores here
We were not sure whether this is a drawback or just peculiarity of QL